doc/BUILD

```diff
@@ -351,7 +353,7 @@ py_test(
 py_test_run_all_subdirectory(
     size = "enormous",
     include = ["external/*.py"],
-    exclude = [],
+    exclude = ["external/test_hashes.py"],
```

Comment:
Is there a reason to put it here when it will then need to be explicitly ignored?

Reply:
it is related to files in that directory?

supposedly files in that directory are external examples, and this file is a signing sheet for changes.